### DISCUSSION TITLE
IOS端chatGPT APP 无法使用问题

### DISCUSSION THREAD

**Discussion by nczyw** [Openwrt-Passwall/openwrt-passwall2#631] [Q&A]
开全局代理，经过多次验证后，发现登陆错误时，提示了本机IPV6信息，也就是说，即使开了全局代理，chatGPT也会使用IPV6去连接，不开全局，连点使用Google登陆这一步点了都报错，感觉passwall2只返回IPV4的设置，漏掉了chatGPT APP上某个域名，导致返回了IPV6地址而出错，路由器上的IPV6对应域名，对我来说查找太难了，有哪位大佬知道，应该如何解决这个问题呢？

**Comment by ghost** [comment]
passwall本身是可以代理IPv6流量的，只要您的代理出口具備相應連接能力，並且本地的WAN可以獲得到IPv6位址，且在網頁介面中选择了 IPv6 透明代理(TProxy) 那就能開啓。
如果上述條件都檢查過，還是不能代理v6流量，則說明您所使用的路由器固件或者設定中存在如下情形：

1.路由器固件中包含docker，其與passwall共存
2.路由器固件中包含其他代理程式，例如clash homeproxy，其與passwall共存
3.路由器固件編譯時沒有選擇必要的防火牆相關組件 （kmod-ipt* iptables* kmod-ip6* ip6tables*）

上面三種情況都是會導致IPv6無法正常代理的原因，如果要解決的話就只能重新編譯相應固件，確保相應防火牆相關組件都選擇上且不存在與passwall衝突的組件。

**Reply by nczyw** [reply]
> passwall本身是可以代理IPv6流量的，只要您的代理出口具備相應連接能力，並且本地的WAN可以獲得到IPv6位址，且在網頁介面中选择了 IPv6 透明代理(TProxy) 那就能開啓。 如果上述條件都檢查過，還是不能代理v6流量，則說明您所使用的路由器固件或者設定中存在如下情形：
> 
> 1.路由器固件中包含docker，其與passwall共存 2.路由器固件中包含其他代理程式，例如clash homeproxy，其與passwall共存 3.路由器固件編譯時沒有選擇必要的防火牆相關組件 （kmod-ipt* iptables* kmod-ip6* ip6tables*）
> 
> 上面三種情況都是會導致IPv6無法正常代理的原因，如果要解決的話就只能重新編譯相應固件，確保相應防火牆相關組件都選擇上且不存在與passwall衝突的組件。

现在的机场出口一般都只有V4地址，具体无法使用的原因，我也只是猜测可能是IPV6直通了，问题肯定是出在passwall2这，IOS我使用手机端的Shadowrocket，chatGPT是正常使用的，前天还发现一个问题，就是B站APP，刷新时，视频封面的评论加载特别慢，我不知道是passwall2设置的dns1.1.1.1的问题，大概要10秒左右，才会加载出来，理论国内域名不应该走国外的DNS服务器的，关闭passwall2后，封面和评论，是瞬间加载出来的，我想，我还是尝试一下homeproxy如何使用的吧。

**Reply by Jackie264** [reply]
> passwall本身是可以代理IPv6流量的，只要您的代理出口具備相應連接能力，並且本地的WAN可以獲得到IPv6位址，且在網頁介面中选择了 IPv6 透明代理(TProxy) 那就能開啓。 如果上述條件都檢查過，還是不能代理v6流量，則說明您所使用的路由器固件或者設定中存在如下情形：
> 
> 1.路由器固件中包含docker，其與passwall共存 2.路由器固件中包含其他代理程式，例如clash homeproxy，其與passwall共存 3.路由器固件編譯時沒有選擇必要的防火牆相關組件 （kmod-ipt* iptables* kmod-ip6* ip6tables*）
> 
> 上面三種情況都是會導致IPv6無法正常代理的原因，如果要解決的話就只能重新編譯相應固件，確保相應防火牆相關組件都選擇上且不存在與passwall衝突的組件。

冒昧问一下：已经是23.05.5了，使用nft并删除了所有legacy，是否也需要这些组件？

**Reply by tankey33** [reply]
IOS端chatgpt在passwall2下现在能使用吗?有解决办法了吗?

**Reply by Jackie264** [reply]
> IOS端chatgpt在passwall2下现在能使用吗?有解决办法了吗?

能用啊 一点没问题

**Reply by OliverX2018** [reply]
我也是开小机场才能上，否则会被识别为china用户。

**Comment by handsomeji** [comment]
是否優先使用IPv6這個是APP、瀏覽器、系統說了算。
你節點出口沒有IPv6，請把远程查询策略調成UseIPv4。或開FakeDNS（直接發域名給服務端去解析。）

**Reply by xmasrain** [reply]
有效，使用了FakeDNS后，重启WiFi，APP可以使用ChatGPT了

**Reply by OliverX2018** [reply]
> 有效，使用了FakeDNS后，重启WiFi，APP可以使用ChatGPT了

我也遇到了同样的问题，但是开了fakeDNS+远端之后还是不行，我的passwall2是作为二级路由的。不知道能不能分享下相关设置，谢谢！

**Reply by Jackie264** [reply]
> > 有效，使用了FakeDNS后，重启WiFi，APP可以使用ChatGPT了
> 
> 我也遇到了同样的问题，但是开了fakeDNS+远端之后还是不行，我的passwall2是作为二级路由的。不知道能不能分享下相关设置，谢谢！

二级路由没试过，不清楚。
但是我在公司使用的单臂旁路由，设置以后通行无阻。

**Comment by OliverX2018** [comment]
恩,今天我把后面高级选项里的的singlebox和xray的东西都勾上,也可以了 

 

发件人: ***@***.*** ***@***.***> 代表 Jackie264
发送时间: 2025年1月9日 15:42
收件人: xiaorouji/openwrt-passwall2 ***@***.***>
抄送: OliverX2018 ***@***.***>; Comment ***@***.***>
主题: Re: [xiaorouji/openwrt-passwall2] IOS端chatGPT APP 无法使用问题 (Discussion #631)

 

有效，使用了FakeDNS后，重启WiFi，APP可以使用ChatGPT了

我也遇到了同样的问题，但是开了fakeDNS+远端之后还是不行，我的passwall2是作为二级路由的。不知道能不能分享下相关设置，谢谢！

二级路由没试过，不清楚。
但是我在公司使用的单臂旁路由，设置以后通行无阻。
